packages/charting/src/chart-type.js

```diff
@@ -13,20 +13,20 @@ const ChartType = withStyles(() => ({
   chartTypeLabel: {
     backgroundColor: 'transparent',
   },
-}))(({ onChange, value, classes }) => (
+}))(({ onChange, value, classes, availableChartTypes, chartTypeLabel }) => (
   <div className={classes.chartType}>
     <FormControl variant={'outlined'} className={classes.chartType}>
       <InputLabel htmlFor="type-helper" className={classes.chartTypeLabel}>
-        ChartType
+        {chartTypeLabel}
       </InputLabel>
 
       <Select value={value} onChange={onChange} labelWidth={0} input={<OutlinedInput name="type" id="type-helper" />}>
-        <MenuItem value={'histogram'}>Histogram</MenuItem>
-        <MenuItem value={'bar'}>Bar Chart</MenuItem>
-        <MenuItem value={'lineDot'}>Line Chart &#9679;</MenuItem>
-        <MenuItem value={'lineCross'}>Line Chart x</MenuItem>
-        <MenuItem value={'dotPlot'}>Dot/Line Plot &#11044;</MenuItem>
-        <MenuItem value={'linePlot'}>Dot/Line Plot X</MenuItem>
+        {availableChartTypes.histogram && <MenuItem value={'histogram'}>{availableChartTypes.histogram}</MenuItem>}
```

Comment:
can you use **availableChartTypes?.histogram** instead, to make sure this doesn't crash?

Reply:
sure!